app-rating/src/main/java/com/stepstone/apprating/AppRatingDialogView.kt

```diff
@@ -204,6 +212,13 @@ class AppRatingDialogView(context: Context) : LinearLayout(context), OnRatingBar
         noteDescriptionText = findViewById(R.id.app_rate_dialog_note_description)
         ratingBar.setIsIndicator(false)
         ratingBar.setOnRatingBarChangeListener(this)
+
+        TextViewCompat.setTextAppearance(titleText, fetchAttributeValue("appRatingDialogTitleStyle"))
```

Comment:
This might be prone to errors if we ever change that. Can't get the value by using `R.attr. appRatingDialogTitleStyle` instead?

Reply:
thats right, corrected